src/app/common/hooks/use-modifier-key.ts

```diff
@@ -8,46 +8,48 @@ export function buildEnterKeyEvent(onClick: () => void) {
   };
 }
 
-let timer = 0;
-
-export function useModifierKey(key: 'alt' | 'control', delay = 0) {
-  const [isPressed, setIsPressed] = useState(false);
-
-  const keydownFn = useCallback(
-    (event: KeyboardEvent) => {
-      if (key === 'alt' && event.altKey) {
-        timer = window.setTimeout(() => setIsPressed(true), delay);
-      }
-      if (key === 'control' && event.ctrlKey) {
-        timer = window.setTimeout(() => setIsPressed(true), delay);
-      }
-    },
-    [delay, key]
-  );
-
-  const keyupFn = useCallback(
-    (event: KeyboardEvent) => {
-      if (key === 'alt' && !event.altKey) {
-        clearTimeout(timer);
-        setIsPressed(false);
-      }
-      if (key === 'control' && !event.ctrlKey) {
-        clearTimeout(timer);
-        setIsPressed(false);
-      }
-    },
-    [key]
-  );
-
-  useEffect(() => {
-    document.addEventListener('keydown', keydownFn);
-    document.addEventListener('keyup', keyupFn);
-    return () => {
-      clearTimeout(timer);
-      document.removeEventListener('keydown', keydownFn);
-      document.removeEventListener('keyup', keyupFn);
-    };
-  }, [keydownFn, keyupFn]);
-
-  return { isPressed };
-}
+// TODO try deprecate this in favour of new settings menu https://github.com/leather-wallet/extension/pull/2732/files
+
+// let timer = 0;
+
```

Comment:
@kyranjamie : what do you think about getting rid of this? I think it's used to show [advanced menu items](https://github.com/leather-wallet/extension/pull/2732/files). 

Maybe we can try replacing it with the radix short keys?

![Screenshot 2024-01-25 at 08 52 25](https://github.com/leather-wallet/extension/assets/2938440/79ba369c-1322-40fe-a525-0e84e9c23121)

Reply:
Not attached to this functionality